### ISSUE TITLE
Without a browser 

### ISSUE THREAD

**Issue by Sz4by** [Ifarra/TikTokBoost#6]
For some reason it doesn't work without a browser, only if I use vpn

**Comment by Ifarra** [comment]
wait I'm confused, so doesn't work without vpn? or with vpn?

**Comment by Sz4by** [comment]
Yes without vpn! If I turn it off, python doesn't load and if I connect a New York vpn it works

**Comment by Sz4by** [comment]
I mean it loads and then gives me a long error and closes

**Comment by Ifarra** [comment]
i see, ill look it up soon, i actually have a plan to implement vpn directly into the script, but i can't find "easy to implement" open vpn yet, so for now, unfortunately you need to use vpn to run the script, thanks for reporting the issue

**Comment by Sz4by** [comment]
You are welcome

**Comment by bledeamarius** [comment]
for me it works from Columbia - with VPN on

**Comment by Sz4by** [comment]
Without a browser?

**Comment by bledeamarius** [comment]
With browser and NordVPN

**Comment by Sz4by** [comment]
It doesn't work for me anymore

**Comment by Sz4by** [comment]
It doesn't work for me anymore 🤣

**Comment by Ifarra** [comment]
😭 I wish i got time to work on this, i've been busy lately, im not touching this project since like for last couple of months

**Comment by Ifarra** [comment]
can you describe what error did you get maybe a screenshot @Sz4by

**Comment by Sz4by** [comment]
> can you describe what error did you get maybe a screenshot @Sz4by

Automatic captcha not working

**Comment by Sz4by** [comment]
And the comments heart You can't set it to send 300 likes to 25 places 🥹
